cfgov/sheerlike/loaders.py

```diff
@@ -1,5 +1,4 @@
 from jinja2.loaders import FileSystemLoader
-import fslib
```

Comment:
This import isn't present in any requirements files and isn't used anywhere, so I've removed it.